src/kernel/src/fs/vnode.rs

```diff
@@ -138,19 +146,22 @@ pub struct VopVector {
     pub default: Option<&'static Self>, // vop_default
     pub access: Option<VopAccess>,      // vop_access
     pub accessx: Option<VopAccessX>,    // vop_accessx
+    pub bypass: Option<VopBypass>,      // vop_bypass
```

Comment:
Have you found a solution for this as I stated in the struct docs?

Reply:
Uh, I completely missed that :sweat_smile:

Reply:
So, just remove it?

Reply:
Yeah unless you have a solution for this.

Reply:
Removed.